<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassesWindow</class>
 <widget class="QMainWindow" name="ClassesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>546</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="addClassButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editClassButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteClassButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goToStudentsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ученики</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadClassesButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Обновить из БД</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveClassesButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string> Сохранить в БД</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <addaction name="actionSchoolReport"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionSchoolReport">
   <property name="text">
    <string>Ведомость всех учеников школы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
